packages/core/src/components/hotkeys/hotkeysTarget.tsx

```diff
@@ -37,12 +43,19 @@ export function HotkeysTarget<T extends { prototype: IHotkeysTarget }>(construct
         this.localHotkeysEvents = new HotkeysEvents(HotkeyScope.LOCAL);
         this.globalHotkeysEvents = new HotkeysEvents(HotkeyScope.GLOBAL);
 
+        if (componentWillMount != null) {
+            componentWillMount.call(this);
+        }
+    };
+
+    // tslint:disable no-invalid-this only-arrow-functions
```

Comment:
Ah, good catch. That was a quick copy/paste and I'm not super familiar with TSLint.